hackathon_site/dashboard/frontend/src/__tests__/Footer.test.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Footer from "../components/general/Footer/Footer";
+
+jest.mock("@material-ui/core/Typography");
+
+describe("Footer", () => {
+    test("renders", () => {
+        const { asFragment } = render(<Footer />);
+
+        expect(asFragment()).toMatchSnapshot();
```

Comment:
I agree - taking out the mock because just wanted to show the other two what's going on with mocking since it will likely be used in our code.

With regards to the date testing - I think that's better than what's there but I don't see too much point in it besides just testing the date, which we would be mocking anyway. I'll go over creating a mock for the date nonetheless when Brenny makes her updates to the date text, and I'll show them a way of mocking the Date natively without having to install `fake-timers`.

Reply:
Apologies, I didn't realize that these tests were just for demonstration purposes! Didn't mean to impose.

If the date is mocked (natively or with sinon), and the mock for Typography is removed so the component fully renders, then a snapshot test would be just as good :+1: